js/tc39/breaking_test_errors.json

```diff
@@ -2697,6 +2688,9 @@
   "test/language/module-code/parse-export-empty.js-strict:true": "test/language/module-code/parse-export-empty.js: ReferenceError: exports is not defined at test/language/module-code/parse-export-empty.js:26:28(3)",
   "test/language/module-code/privatename-valid-no-earlyerr.js-strict:true": "test/language/module-code/privatename-valid-no-earlyerr.js: SyntaxError: test/language/module-code/privatename-valid-no-earlyerr.js: Unexpected character '#' (33:2)\n  31 | \n  32 | class outer {\n\u003e 33 |   #x = 42;\n     |   ^\n  34 | \n  35 |   f() {\n  36 |     var self = this; at \u003ceval\u003e:2:28542(114)",
   "test/language/reserved-words/await-module.js-strict:true": "test/language/reserved-words/await-module.js: error is not an object (Test262: This statement should not be evaluated.)",
+  "test/language/reserved-words/ident-reference-false.js-strict:true": "test/language/reserved-words/ident-reference-false.js: error is not an object (Test262: This statement should not be evaluated.)",
+  "test/language/reserved-words/ident-reference-null.js-strict:true": "test/language/reserved-words/ident-reference-null.js: error is not an object (Test262: This statement should not be evaluated.)",
+  "test/language/reserved-words/ident-reference-true.js-strict:true": "test/language/reserved-words/ident-reference-true.js: error is not an object (Test262: This statement should not be evaluated.)",
```

Comment:
`falce`, `null` and `true` should not be usable as identifiers

Reply:
Is this a goja bug then? And same question for `this`?

Reply:
seems so, some checking is not done anymore in order to allow ... some other keywords to be used as identifiers in some cases namely when they are escaped as properties of objects, see https://github.com/dop251/goja/commit/13aee8e3cc3dad5437c2dea65c534d4bf2afb11e